src/topp/CometAdapter.cpp

```diff
@@ -533,6 +570,13 @@ class TOPPCometAdapter :
     os << "output_suffix =\n";                                   // add a suffix to output base names i.e. suffix "-C" generates base-C.pep.xml from base.mzXML input
     os << "mass_offsets = " << ListUtils::concatenate(getDoubleList_("mass_offsets"), " ") << "\n"; // one or more mass offsets to search (values subtracted from deconvoluted precursor mass)
     os << "precursor_NL_ions =\n"; //  one or more precursor neutral loss masses, will be added to xcorr analysis 
+    os << "pinfile_protein_delimiter = " << getStringOption_("pinfile_protein_delimiter") << "\n";
```

Comment:
would this also work with the way you did it before (empty string)?

Reply:
If this parameter is left blank or is missing, the default tab delimiter is used.